gtfs-realtime/spec/en/reference.md

```diff
@@ -634,16 +641,45 @@ _Propagated detour delays affect all stops following the end of a modification.
 
 | _**Field Name**_ | _**Type**_ | _**Required**_ | _**Cardinality**_ | _**Description**_ |
 |------------------|------------|----------------|-------------------|-------------------|
-| **start_stop_sequence** | [uint32](https://protobuf.dev/programming-guides/proto2/#scalar) | Required | One | The stop sequence value from the (CSV) GTFS of the first stop of the original trip that is to be affected by this modification. |
-| **num_stops_replaced** | [uint32](https://protobuf.dev/programming-guides/proto2/#scalar) | Required | One | The number of stops which are canceled and replaced by the `replacement_stops`. May be zero to indicate no skipped stops. |
+| **start_stop_selector** | [StopSelector](#message-stopselector) | Required | One | The stop selector of the first stop of the original trip that is to be affected by this modification. If `end_stop_selector` is not provided, no stop will be removed and the stop from the `start_stop_selector` is only provided as a reference stop for `travel_time_to_stop` calculations. |
+| **end_stop_selector** | [StopSelector](#message-stopselector) | Conditionally required | One | The stop selector of the last stop of the original trip that is to be affected by this modification. The selection is inclusive, so if only one stop_time is replaced by that modification, `start_stop_selector` and `end_stop_selector` should be equivalent. If no stop_time is replaced, `end_stop_selector` should not be provided. It's otherwise required.  |
```

Comment:
If you do end up keeping the above "If `end_stop_selector` is not provided, no stop will be removed and the stop from the `start_stop_selector` is only provided as a reference stop for `travel_time_to_stop` calculations." I would suggest being more consistent with the language "no stop will be removed" vs. "no stop_times replaced"